### PR TITLE
switch config a and b

### PR DESCRIPTION
**Pull request by pavlenex**
This PR puts an emphasis on configuration with tProxy.

### CODE REVIEW

**Comment by pavlenex** [comment]
As discussed in #184 

@GitGab19 pleasereview for technical accuracy, I'm sure we can still iron out the copy, but wanted to kick-off efforts.

File: src/index.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ configurationTabs:
     value: config_A
     visualSrc: /assets/config-a.svg
     mobileVisualSrc: /assets/config-a-mobile.svg
-    textContent: Before Stratum V2, transaction sets were selected by pools. With this SV2 configuration they’re selected by individual miners, making the network more censorship-resistant. Miners run SV2 firmware, connecting to the SV2 proxy server. Miners can pick their transactions locally and declare them to an SV2 pool.
+    textContent: Miners can utilize Stratum v2 without updating to SV2 firmware. Connect your SV1 firmware devices through Translation Proxy which facilitates the conversion of SV1 messages to SV2 for communication with an SV2 pool. Miners handle transaction selection locally via their own Template Provider, and declare them to a pool. If a pool rejects the proposed transactions, miners automatically switch to an alternative pool. Should all configured pools decline the proposal, miners fallback to solo mining.
```
Comment (GitGab19) [inline src/index.md:31]: Do we want to use Sv2/v1 instead of SV2/V1 everywhere?

File: src/index.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ configurationTabs:
     value: config_D
     visualSrc: /assets/config-d.svg
     mobileVisualSrc: /assets/config-d-mobile.svg
-    textContent: In this configuration miners run mining devices with SV1 firmware and connect to the translation proxy which translates the SV1 to SV2 messages and sends them to the SV2 pool. Transaction selection is done by the miners locally and then declared to the pool.
+    textContent: In this configurations, mining pool select transactions, but it uses all the security and performance features of the SV2 protocol.
```
Comment (GitGab19) [inline src/index.md:46]: - configuration
- selects

**Review by GitGab19 — COMMENTED** [review]
Still not completely sure about the best configs order: should we place config C (Translator+JDP) as config B?
Since the only configurations which can be test with real asic machines are the ones with the Translator (?)
What do you think about it?

File: src/index.md (hunk 1 above)
Comment (pavlenex) [reply src/index.md:31]: Yeah Stratum v2 so should be  Sv2 if that's what you mean?

Comment (GitGab19) [reply src/index.md:31]: > Yeah Stratum v2 so should be Sv2 if that's what you mean?

Yeah exactly 👍

Comment (pavlenex) [reply src/index.md:31]: Based on recent discussions decided to uppercase everything for now both Stratum V2 and SV2, will update based on discussion in #193

File: src/index.md (hunk 2 above)
Comment (pavlenex) [reply src/index.md]: Fixed https://github.com/stratum-mining/stratumprotocol.org/pull/188/commits/d9735488f1115a56513c858e28129fb5628475eb

**Comment by pavlenex** [comment]
> Still not completely sure about the best configs order: should we place config C (Translator+JDP) as config B? Since the only configurations which can be test with real asic machines are the ones with the Translator (?) What do you think about it?

I am unsure either, however at some point (perhaps sooner than we think) we need to talk about SV2 firmware, and I kinda think it's okay to emphasize that as a possible option, though currently it's not possible, I also wanted to emphasize the configuration where JD is used, since that's the biggest selling point of SV2 for most people.

I think this one is good to go now, I'll leave it up to you to make a call.

**Review by GitGab19 — APPROVED** [review]
I agree with you on the configuration order.
LGTM

**Pull request merged**